bangazonapi/views/product.py

```diff
@@ -276,6 +277,14 @@ def sold_filter(product):
 
             products = filter(sold_filter, products)
 
+        if min_price is not None:
+            def min_price_filter(product):
+                if product.price >= int(min_price):
```

Comment:
what if the minimum price is $11,000.13 ?

Reply:
I've updated the code to change the typecast to a `float` instead of an `int`.  Also updated the test to validate that float prices are supported.